integrations/langfuse/pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
 dependencies = [
-  "haystack-ai",
+  "haystack-ai>=2.1.0",
```

Comment:
It was because of .to_openai_format(). Which was not existing for haystack < 2.1 https://github.com/deepset-ai/haystack/commit/c789f905bc0021be87b04e2d5e9c319d05dedee1#diff-3df0900d8e6a62e1299f3df22daa4957bba9a7d00a625d50ab0129537dac91f3R31